### ISSUE TITLE
AndroidX migration

### ISSUE THREAD

**Issue by Darex1991** [henninghall/react-native-date-picker#683]
Hi, yesterday I ran migration from Android studio from Jetifier to Androidx and it changed react-native-date-picker. What do you think? It is necessary or useless?

patch for 4.1.1:
```
diff --git a/node_modules/react-native-date-picker/android/build.gradle b/node_modules/react-native-date-picker/android/build.gradle
index f1cc0a2..f2c3304 100644
--- a/node_modules/react-native-date-picker/android/build.gradle
+++ b/node_modules/react-native-date-picker/android/build.gradle
@@ -12,7 +12,7 @@ android {
         minSdkVersion safeExtGet('minSdkVersion', 18)
         targetSdkVersion safeExtGet('targetSdkVersion', 25)
 
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
     }
     buildTypes {
         release {

```
Thanks

**Comment by henninghall** [comment]
Thanks for the question, since androidx is working from compile sdk version 28, (RN 0.58), we could probably start using it now and drop support for RN 0.57. I think that is fine.

**Comment by Darex1991** [comment]
Thanks for response. You think before migrating to RN 0.58 my project would no blow up if I use this patch :)?

**Comment by henninghall** [comment]
The patch should not actually be needed since this is how jetifier works, it modifies the code on the fly, before you compile it. No need to patch.

**Comment by Darex1991** [comment]
Hmm but without it when I run `checkJetifier` I've got error about needed migration to androidX, or am I interpreting it wrong?

**Comment by henninghall** [comment]
Jetifier has been included as a build step in react native android by default since a while back. It is controlled by `enableJetifier` in the build.gradle file. So no need to run it manually. Maybe I am not understanding your issue correctly?

**Comment by Darex1991** [comment]
Hi, sorry for late response - holidays :)
Now, I'm little confused. I saw a tutorial about migrating to androidX https://youtu.be/Hyt7LR5mXLc?si=V4CUT_q8fhNkk6jN&t=472 that I should also change 3rd dependencies. I created patches to not do "Migrate to AndroidX" on all users and on appcenter I can not do that (because it is doing just locally). If I don't need to, than what is for?
 
Sorry, more questions than answers in my head. Maybe I just understand that wrong. @henninghall
